github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml

```diff
@@ -278,6 +309,38 @@ postsubmits:
               limits:
                 memory: "16Gi"
                 cpu: "500m"
+    - name: publish-vm-image-builder-image
+      always_run: false
+      run_if_changed: "images/vm-image-builder/.*"
+      annotations:
+        testgrid-create-test-group: "false"
+      decorate: true
+      max_concurrency: 1
+      labels:
+        preset-dind-enabled: "true"
+        preset-kubevirtci-quay-credential: "true"
+      cluster: phx-prow
+      spec:
+        nodeSelector:
+          type: vm
+        containers:
+          - image: quay.io/kubevirtci/bootstrap:v20210906-994b913
+            command:
+              - "/usr/local/bin/runner.sh"
+              - "/bin/bash"
+              - "-c"
+              - |
+                cat "$QUAY_PASSWORD" | docker login --username $(cat "$QUAY_USER") --password-stdin=true quay.io
+                cd images
+                ./publish_image.sh vm-image-builder quay.io kubevirtci
```

Comment:
I can't find the `vm-image-builder` directory in project-infra `images/`. Is it supposed to be somewhere else?

Reply:
It will be added with this PR, we didn't have it before.